README.md

```diff
@@ -127,7 +127,7 @@ Here are a few examples for throwing your own exception message.
 ```csharp
 if (!ModelState.IsValid)
 {
-    throw new ApiException(ModelState.AllErrors());
+    throw new ApiException(ModelState);
 }
```

Comment:
`ApiException` accepts the type of `IEnumerable<ValidationError>` so it should be:

```csharp
throw new ApiException(ModelState.AllErrors());
```

Reply:
I'm sorry I was carelesso (╥﹏╥)o

Reply:
PR merged! Thank you! :)